mdpow/fep.py

```diff
@@ -244,16 +246,24 @@ def load(cfg, section):
         def getter(type, section, key):
             try:
                 return cfg_get[type](section, key)
-            except ConfigParser.NoOptionError:
+            except NoOptionError:
                 return None
         return FEPschedule((key, getter(keytype, section, key)) for key,keytype in keys.items()
                            if getter(keytype, section, key) is not None)
 
     def __deepcopy__(self, memo):
```

Comment:
Not sure, it is correctly imported in 2.7 and 3.x thought.

Reply:
A test that covers it would be good.

Reply:
According to https://app.codecov.io/gh/Becksteinlab/MDPOW/compare/166/changes#D1L474 you removed a use of `__deepcopy__` because it actually failed. Thus, a test is really necessary and perhaps then you find a way to revert the workaround and use `__deepcopy__` again.

Reply:
This needs a test.

Reply:
I got at test to hit that according to the pycharm coverage report